src/util/getTopologicalOrder.ts

```diff
@@ -3,32 +3,36 @@ export interface Edge {
 	target: string;
 }
 
-// Uses the Khan Algorithm loosely
+/**
+	* Uses the Khan Algorithm (this implementation cannot work with isolated nodes)
+	* @param {Set<Edge>} graph - A graph represented as set of edges
+	* @return {Array<string>} The topologial oder of the graph
+	*/
 export default function getTopologicalOrder(inputGraph: ReadonlySet<Edge>): string[] {
 	const graph: Set<Edge> = new Set(inputGraph);
 
 	const order: string[] = [];
-	const vertexesWithNoInEdge: Set<string> = getVertexesWithNoInEdge(graph);
+	const nodesWithNoInEdge: Set<string> = getNodesWithNoInEdge(graph);
 
 	// These are to avoid issues described in following thread by externalizing deletions (from the loop):
 	// https://github.com/diproche/webinterface/pull/28/files/4647fd1d5b3e9050a532c5c24f98d14643f1661e#r287705437
 	const edgesToRemove: Set<Edge> = new Set();
-	const vertexesToRemove: Set<string> = new Set();
+	const nodesToRemove: Set<string> = new Set();
 
- while (vertexesWithNoInEdge.size !== 0) {
-		vertexesWithNoInEdge.forEach((vertex: string) => {
-			vertexesToRemove.add(vertex);
-			order.push(vertex);
+ while (nodesWithNoInEdge.size !== 0) {
+		nodesWithNoInEdge.forEach((node: string) => {
+			nodesToRemove.add(node);
+			order.push(node);
 			graph.forEach((edge: Edge) => {
-					if (edge.origin === vertex) { edgesToRemove.add(edge); }
-					vertexesWithNoInEdge.add(edge.target);
+					if (edge.origin === node) { edgesToRemove.add(edge); }
+					nodesWithNoInEdge.add(edge.target);
 				});
 			// Externalized Deletion
 			edgesToRemove.forEach((edge: Edge) => graph.delete(edge));
 		});
 		// Externalized Deletion
-		vertexesToRemove.forEach((toRemove: string) => vertexesWithNoInEdge.delete(toRemove));
-		vertexesToRemove.clear();
+		nodesToRemove.forEach((toRemove: string) => nodesWithNoInEdge.delete(toRemove));
+		nodesToRemove.clear();
 	}
 
 	// If there are cycles in the graph the residual graph will be returned (which contains the cycles)
```

Comment:
This comment is redundant. Also, `graph.toString` is a function, not a `string`. You need to invoke it (add `()`).